src/main/java/game/applicationBase/LocalFileTool.java

```diff
@@ -52,13 +73,16 @@ public static Game makeNewGameFromFile(ClassLoader classLoader, File file) throw
         JsonReader actionStoreReader = new JsonReader(new FileReader(mainActionsFile));
         JsonReader itemStoreReader = new JsonReader(new FileReader(mainItemFile));
         JsonReader sceneStoreReader = new JsonReader(new FileReader(mainSceneFile));
+        JsonReader characterStoreReader = new JsonReader(new FileReader(mainCharactersFile));
 
         SceneStore sceneStore = gson.fromJson(sceneStoreReader, SceneStore.class);
         ItemStore itemStore = gson.fromJson(itemStoreReader, ItemStore.class);
         ActionStore actionStore = gson.fromJson(actionStoreReader, ActionStore.class);
+        CharacterStore characterStore = gson.fromJson(characterStoreReader, CharacterStore.class);
 
         Game game = new Game(sceneStore.toIntegerSceneMap(),
-                actionStore.toIntegerActionMap(), itemStore.toStringItemMap(), sceneStore.getStartSceneId());
+                actionStore.toIntegerActionMap(), itemStore.toStringItemMap(),
+                characterStore.toStringCharacterMap(), sceneStore.getStartSceneId());
 
         return game;
     }
```

Comment:
I can see some changes inside the file handling class, however, the file management is still great! Nice work!